.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            2
+        ],
+        "quotes": [
+            1,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ]
+    },
+    "env": {
+        "node": true
+    },
+    "extends": "eslint:recommended"
```

Comment:
Add the following rules:

* `no-extra-parens`
* `curly`
* `default-case`
* `dot-location`, property
* `dot-notation`
* `eqeqeq` smart
* `no-caller`
* `no-else-return`
* `no-empty-label`
* `no-empty-pattern`
* `no-eval`
* `no-extend-native`
* `no-extra-bind`
* `no-floating-decimal`
* `no-implied-eval`
* `no-invalid-this`
* `no-labels`
* `no-lone-blocks`
* `no-loop-func`
* `no-magic-numbers`
* `no-native-reassign`
* `no-new-func`
* `no-new-wrappers`
* `no-new`
* `no-param-reassign`, props true
* `no-process-env`, make sure warning for now
* `no-proto`
* `no-return-assign`
* `no-script-url`
* `no-self-compare`
* `no-sequences`
* `no-unused-expressions`, enable allowShortCircuit and allowTernary
* `no-useless-call`
* `no-useless-concat`
* `no-void`
* `no-with`
* `radix`
* `yoda` never
* `init-declarations` always
* `no-undef-init`
* `no-undefined`
* `no-use-before-define`
* `callback-return`
* `global-require`, leave as warning
* `handle-callback-err`
* `no-mixed-requires`, grouping true
* `no-new-require`
* `no-path-concat`
* `array-bracket-spacing` never
* `block-spacing` always
* `brace-style` 1tbs
* `camelcase` always
* `comma`, before false, after true
* `comma-style` last
* `eol-last`
* `func-style` expression
* `key-spacing`, use defaults
* `new-cap`
* `new-parens`
* `newline-after-var` always
* `no-array-constructor`
* `no-continue`
* `no-lonely-if`
* `no-multiple-empty-lines`
* `no-nested-ternary`
* `no-new-object`
* `no-restricted-syntax`, ClassDeclaration, ClassExpression
* `no-spaced-func`
* `no-trailing-spaces`
* `no-underscore-dangle`
* `no-unneeded-ternary`
* `object-curly-spacing` always
* `one-var` never
* `operator-linebreak` after
* `quote-props` as-needed
* `quotes` backtick, leave as warning until our ES6 transition is complete
* `require-jsdoc`
* `semi-spacing`, use the default options
* `space-after-keywords` always
* `space-before-blocks` always
* `space-before-function-paren` never
* `space-before-keywords` always
* `space-in-parens` never
* `space-infix-ops`
* `space-return-throw-case`
* `space-unary-ops`, use defaults
* `spaced-comment` always
* `arrow-parens` as-needed
* `arrow-spacing` use defaults
* `no-const-assign`
* `prefer-arrow-callback`
* `prefer-template`
* `comma-dangle` never

Things to try, but may cause issues:
* `valid-jsdoc`
* `no-multi-spaces`
* `no-multi-str`
* `no-sync`
* `no-negated-condition`
* `padded-blocks` never

:sweat_smile: